docker-compose.search.yml

```diff
@@ -0,0 +1,95 @@
+version: "3.9"
+
+services:
+  search:
+    image: ghcr.io/nashtech-garage/yas-search:latest
+    entrypoint: [ "./wait-for-it.sh", "elasticsearch:9200", "--timeout=300", "--", "java", "-jar", "/search-0.0.1-SNAPSHOT.jar" ]
+    environment:
+      SERVER_SERVLET_CONTEXT_PATH: /search
+      YAS_PUBLIC_URL: http://api.yas.local/search
+      YAS_SERVICES_PRODUCT: http://product/product
+      SERVER_PORT: "80"
+    depends_on:
+      - kafka
+    networks:
+      - yas-network
+  zookeeper:

```

Comment:
zookeeper, kafka, kafka connect, akhq not just be used to feed data for elastic search. I think we should put them to the core services (docker-compose.yml)

Reply:
Currently, kafka-connect depends on elasticsearch. Let me try kafka-connect still works without depending on elasticsearch.

Reply:
I moved them to the core services.